python/cuml/test/test_kneighbors_classifier.py

```diff
@@ -44,13 +44,13 @@ def _build_train_test_data(X, y, datatype, train_ratio=0.9):
     y_test = y[~train_selection]
 
     if datatype == "dataframe":
-        X_train = cudf.DataFrame.from_gpu_matrix(
+        X_train = cudf.DataFrame(
```

Comment:
Are `to_device(X)` calls always necessary, or will `cudf.DataFrame(X)` work even with a host input?

Reply:
Indeed they are not needed, just went ahead and removed all of those